dm/tests/dmctl_basic/conf/get_master1.toml

```diff
@@ -20,10 +20,10 @@ max-request-bytes = 1572864
 auto-compaction-mode = "periodic"
 auto-compaction-retention = "1h"
 quota-backend-bytes = 2147483648
+openapi = false
 v1-sources-path = ""
 ssl-ca = ""
 ssl-cert = ""
 ssl-key = ""
 
 [experimental]
-  openapi = false
```

Comment:
if experimental.openapi==true, still print it? Why not set it to false when adjust?

Reply:
yes we should adjust this value , added in https://github.com/pingcap/tiflow/pull/3836/commits/09647710a2a3d6b8acf5df886ce629dded1c786a